pkg/lib/userexport/model_test.go

```diff
@@ -65,5 +65,40 @@ func TestRequest(t *testing.T) {
 		`, `invalid request body:
 /format: enum
   map[actual:unknown format expected:[ndjson csv]]`)
+
+		test(`
+{
+	"format": "csv",
+	"csv": {
+		"fields": []
+	}
+}
+		`, "")
```

Comment:
This test is good. But the result is wrong. If fields is specified, then it must be an array with at least one item. 

https://github.com/authgear/authgear-server/pull/4717/files#diff-5d0c564f0dbed14ca1655b1752d72835c8115dbe937c0a20f830584f1ff8edefR59

> If this is specified, then it must be an non-empty list.

So your fix is add `minItems: 1`

Reply:
added this condition